src/openforms/submissions/views.py

```diff
@@ -140,11 +140,14 @@ class ResumeSubmissionView(ResumeFormMixin, RedirectView):
 
     def get_form_resume_url(self, submission: Submission) -> str:
         form_resume_url = furl(submission.form_url)
+
+        state = submission.load_execution_state()
+        last_completed_step = state.get_last_completed_step()
+        target_step = last_completed_step or state.submission_steps[0]
+
```

Comment:
found out this was broken while adding tests - if no step was completed yet, this didn't work.